src/components/projects/ProjectDetail.tsx

```
@@ -1,164 +1,168 @@
+          onClose={() => setShowEditTask(null)}
+          onSave={async (taskId, updates) => {
+            await updateTask(taskId, updates);
+            setShowEditTask(null);
+          }}
+        />
+      )}
+       {showEditProject && (
+        <EditProjectModal
+          project={showEditProject}
+          onClose={() => setShowEditProject(null)}
+          onSave={async (id, updates) => {
```

ESLint

[error] 'id' is defined but never used.

(@typescript-eslint/no-unused-vars)